media-source-respec.html

```diff
@@ -2431,45 +2389,34 @@
           <p>The type of track that can be constructed using the information in this object.</p>
-        </dd>
-        <dt>readonly attribute DOMString byteStreamTrackID</dt>
-        <dd>
+        </dd><dt><dfn><code>byteStreamTrackID</code></dfn> of type <span class="idlAttrType"><a>DOMString</a></span>, readonly       </dt><dd>
           <p>The decimal string representation of the <a def-id="track-id"></a> that identifies the specific
             track that this object SHOULD apply to. An empty string indicates that this object can be
             used for any track of the appropriate type (e.g., any <a>AudioTrack</a> if
             <a def-id="TrackDefault-type"></a> equals <a def-id="TrackDefaultType-audio"></a>).</p>
-        </dd>
-        <dt>readonly attribute DOMString language</dt>
-        <dd>
+        </dd><dt><dfn><code>language</code></dfn> of type <span class="idlAttrType"><a>DOMString</a></span>, readonly       </dt><dd>
           <p>The default language to use when an <a def-id="init-segment"></a> does not contain language
           information for a new track. See <a def-id="init-segment-received-algorithm"></a> for details.</p>
-        </dd>
-        <dt>readonly attribute DOMString label</dt>
-        <dd>
+        </dd><dt><dfn><code>label</code></dfn> of type <span class="idlAttrType"><a>DOMString</a></span>, readonly       </dt><dd>
           <p>The default label to use when an <a def-id="init-segment"></a> does not contain label
           information for a new track. See <a def-id="init-segment-received-algorithm"></a> for details.</p>
-        </dd>
-        <dt>sequence&lt;DOMString&gt; getKinds()</dt>
-        <dd>
+        </dd></dl></section><section><h2>Methods</h2><dl class="methods" data-dfn-for="TrackDefault" data-link-for="TrackDefault"><dt><dfn><code>getKinds</code></dfn></dt><dd>
           <p>The default kinds used when an <a def-id="init-segment"></a> does not contain kind
           information for a new track. See <a def-id="init-segment-received-algorithm"></a> for details.</p>
-        </dd>
-      </dl>
+        <div><em>No parameters.</em></div><div><em>Return type: </em><code>sequence&lt;<a>DOMString</a>&gt;</code></div></dd></dl></section></div>
     </section>
 
     <section id="trackdefaultlist">
       <h2>TrackDefaultList Object</h2>
       <p>TrackDefaultList is a simple container object for <a>TrackDefault</a> objects. It provides read-only array access.</p>
 
-      <dl title="interface TrackDefaultList" class="idl">
-        <dt class="extended-attribute">
-          Constructor(optional sequence&lt;TrackDefault&gt; trackDefaults = [])
-        </dt>
-        <dd>
+      <div><pre class="idl">[ Constructor (optional sequence&lt;TrackDefault&gt; trackDefaults = [])]
+interface TrackDefaultList {
```

Comment:
Should this be linked to the TrackDefaultList constructor?  Seems not, and it isn't in the current WD.